content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -0,0 +1,359 @@
+---
+title: Enabling Rate Limits using Envoy
+description: This task shows you how to use Istio to dynamically limit the traffic to a service.
+weight: 10
+keywords: [policies,quotas]
+owner: istio/wg-policies-and-telemetry-maintainers
+test: n/a
```

Comment:
```suggestion
test: no
```

Reply:
Note that a test is required for new docs but if you want, we can merge this without it and add the test in a followup PR: https://github.com/istio/istio.io/blob/master/tests/README.md#test-authoring-overview